workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Dropdown/Dropdown.tsx

```diff
@@ -27,8 +27,8 @@ const DropDown = ({className, closeOnClick, icon, label, options}: Props) => (
 		className={className}
 		closeOnClick={closeOnClick}
 		trigger={
-			<ClayButton aria-label="Action Button" displayType="unstyled">
-				{label && (
+			<ClayButton displayType="unstyled">				
```

Comment:
remove the unneeded tabs or spaces here after `<ClayButton displayType="unstyled">`

Reply:
@kylembischof to make things easier I used this PR to push the bug fixes to PROD, this specific change was already in master, but was not suppose to go to PROD yet, it is related to [ISSD-2137](https://liferay.atlassian.net/browse/ISSD-2137)